apps/prairielearn/src/pages/instructorAssessmentSettings/instructorAssessmentSettings.html.ts

```diff
@@ -74,15 +85,9 @@ export function InstructorAssessmentSettings({
                 </div>
                 <div class="form-group">
                   <label for="set">Set</label>
-                  <input
-                    type="text"
-                    class="form-control"
-                    id="set"
-                    name="set"
-                    value="${resLocals.assessment_set.name} (${resLocals.assessment_set
-                      .abbreviation})"
-                    disabled
-                  />
+                  <select class="form-control" id="set" name="set">
+                    ${setOptions({ assessmentSets, selectedSet: resLocals.assessment_set.name })}
```

Comment:
Why this separate functions? It's a small amount of code and is only ever used in one place. Ditto for `modulesOptions`.

Reply:
Moved these so they are no longer separate functions in [ca2ca5d](https://github.com/PrairieLearn/PrairieLearn/pull/10677/commits/ca2ca5d0e50c8f76d05e06333bbe2e8d3c49cf1c).